CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1,5 @@
 Refresh Documentation
 Add NGSIv2 metadata support to device provisioned attributes
 Fix: Error message when sending measures with unknown/undefined attribute
-Add Null check within executeWithSecurity() to avoid crash (#829)
\ No newline at end of file
+Add Null check within executeWithSecurity() to avoid crash (#829)
+Add NGSIv2 metadata support to attributeAlias plugin.
```

Comment:
Should the documentation in https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/advanced-topics.md#metadata-support be modified due to this PR?

Not sure if this PR involves new functionality  (so the documentation should be probably updated) or is covering a gap in existing functionality (in which case probably documentation doesn't need to be updated)

Reply:
Already covered in https://github.com/telefonicaid/iotagent-node-lib/blob/feature/add-v2-metadata/doc/advanced-topics.md#metadata-support

Reply:
NTC